### DISCUSSION TITLE
Form for multi-sites

### DISCUSSION THREAD

**Discussion by minalove** [solspace/craft-freeform#223] [Questions & Answers] [answered]
Hi,

Can I define different recipients for a form depending on the current site in a multi-site setup?

Kind regards,
Mina

**Comment by kjmartens** [comment, marked as answer]
Inside the form builder, there isn't a way to configure site-specific details. The best way to handle this across multiple sites would be to make use of the template-level `dynamicNotification` parameter approach. It will allow you to set the email notification and notification template inside the page template, allowing you to set unique values. 🙂 

https://docs.solspace.com/craft/freeform/v3/overview/email-notifications.html#dynamic-template-notifications

**Reply by sm9** [reply]
Hi Kelsey, I was just going to ask a similar question, so good to see this question already asked and answered. I have a follow-up question for this approach - in Freeform Pro, when choosing the option to 'Resend Notifications', or 'Send Additional Notification', do either of these options respect the template-level dynamicNotification parameter that was used when sending the original notification, to re-send the notification (or additional notification)? We have a multi-site where each individual site will send a notification email to a dynamic recipient when it is originally submitted, but we're not sure whether this will also be used when re-sending notifications (as we can't see this value stored anywhere). Thanks, Stephen

**Reply by kjmartens** [reply]
Good question @sm9. In the case of Dynamic Notifications (at template level), Freeform will not be aware of these, so the Resend Notifications feature will not include that.

The **Send Additional Notification** feature, however, is different. It will ignore any existing email notifications configured for the form but will route the submission data to the email address that you enter into the **Send Additional Notification** feature area. Does this make sense? 🙂

**Reply by sm9** [reply]
Hi Kelsey, that makes sense, thanks, so we'll use that approach when needed. Thanks! :)
